### DISCUSSION TITLE
copilot4eclipse chat hangs if eclipse has been idle for a while

### DISCUSSION THREAD

**Discussion by davidmichaelkarr** [Genuitec/Copilot4Eclipse-community#34] [Help]
I'm using version 1.2.1.202406240349 of copilot4eclipse, on Windows 11. Me and some of my coworkers are seeing a symptom where even though copilot says "Ready", when we enter a question in the chat window and press enter, we get the three dots slowly changing, but it goes on forever. It never returns an answer. However, if we restart Eclipse and reenter the question, it returns the answer. I don't know how long it has to be idle before it gets into this state.

**Comment by wayneparrott** [comment]
Thanks for the report. 

A couple of questions:
1. Do you recall if the machine hibernated or other power-saving profile (e.g., turn off network adapters after some time) was in effect? 
2. When Chat is not responding are Copilot code completions in an editor being generated as expected?

Wayne
Copilot4Eclipse Team
